templates/init/functions/javascript/eslint.json

```diff
@@ -1,7 +1,7 @@
 {
   "parserOptions": {
     // Required for certain syntax usages
-    "ecmaVersion": 6
+    "ecmaVersion": 8
```

Comment:
Is this actually right? I don't think `ecmaVersion` corresponds to Node.js version. There is ES6, but there is no ES8

Reply:
Huh, [it is](https://eslint.org/docs/user-guide/configuring):

> ecmaVersion - set to 3, 5 (default), 6, 7, 8, 9, or 10 to specify the version of ECMAScript syntax you want to use. You can also set to 2015 (same as 6), 2016 (same as 7), 2017 (same as 8), 2018 (same as 9), or 2019 (same as 10) to use the year-based naming.

That said, personally I'd stick with `es2015` (or `6`) as that's supported by Node `8` just fine. (also, sticking with `esYYYY` to match `tsconfig.json`)

Reply:
Let's change this to es2017 since we're upgrading everything to 8.

Reply:
+1 to es2017 since that includes async/await